src/components/shared/AboutDialog/AboutDialog.tsx

```diff
@@ -21,10 +24,13 @@ const AboutDialog: FC<AboutDialogProps> = ({open, onClose}) => {
                     Hours was created by Nathan Luu, Nick Young, and Neil Ramaswamy at Brown University to facilitate
                     office hours for courses in the CS Department and beyond.
                 </Typography>
-                {/*<Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined"*/}
-                {/*        href="https://github.com/nthnluu/hours-frontend/wiki/Contributing">*/}
-                {/*    Contribute*/}
-                {/*</Button>*/}
+                <Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined" href={"https://github.com/nthnluu/hours-frontend/wiki/Contributing"}>
+                   Contribute
+                </Button>
+                <Divider/>
+                <Typography>
```

Comment:
Can we hyperlink this to https://cs.brown.edu/people/memoriam/william-povell/ as is done in SignMeUp right now?